microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -340,6 +340,14 @@ Collecting child browsing data and sharing it with the family group organizer re
 The family group organizer may stop the data collection from the family safety portal.  For more information about Microsoft family safety features, see [What is a Microsoft family group?](https://support.microsoft.com/help/12413)
 
 
+<!-- ====================================================================== -->
+## Favorites
+
+Favorites now supports views that add an image thumbnail for favorites that you've saved.  All favorites data is stored locally on the device, organized per Microsoft Edge profile.  Favorites are saved for the duration the favorite exists (10 years).  If you have sync turned on for favorites, your saved items are available across all signed-in and syncing versions of Microsoft Edge.
+
+If you want Microsoft Edge not to capture images for your favorites, go to `edge://settings/privacy`, scroll down to the **Services** section, and then turn off the toggle for **Save Images for Favorites**.<!-- todo: no such control found in Canary 117 -->  All previously saved images will be erased.
```

Comment:
What's the label on the toggle?  I'd expect lowercase 'images'.  
Which Edge version will have the toggle?

In Canary 117, in `edge://settings/privacy` > **Services** section, **Save Images for Favorites** is not found.

My Canary 117 > Settings > Services (can't guarantee these are default toggle settings):
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/0c5d5d24-0375-46f3-aa50-ca575e9d6204)

Reply:
The new Favorites section was removed from this PR and moved to a dedicated PR 2771.  The rendered Favorites section in that PR
https://review.learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-2771#favorites
has latest planned label: "turn off the toggle for **Save images for Favorites**."